vpr/src/route/route_common.cpp

```diff
@@ -281,7 +283,9 @@ bool try_route(int width_fac,
                     router_opts.trim_obs_channels,
                     router_opts.clock_modeling,
                     directs, num_directs,
-                    &warning_count);
+                    &warning_count,
+                    router_opts.read_edge_metadata,
+                    router_opts.do_check_rr_graph);
```

Comment:
The argument list of create_rr_graph is getting long. Since many of those options are coming from router_opts, it may make sense to just pass the whole router_opts in directly.

Reply:
I'm a fan of keeping things explicit.  If we really need to do this, I believe we should split the rr generator options from the router options, so we can more easily track which of the options are actually used by `create_rr_graph`.